### PR TITLE
FreeBSD with Contextulization

### PR DESCRIPTION
**Pull request by perrymanuk**
Would like to contribute our FreeBSD image from Trivago to the marketplace

### CODE REVIEW

File: appliances/Trivago/ebb81aaf055247c19579723cc84b8e74.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+id: ebb81aaf055247c19579723cc84b8e74
```
Comment (vholer) [inline appliances/Trivago/ebb81aaf055247c19579723cc84b8e74.yaml]: Can you please remove the **id** from YAML, it's taken from the filename...
I have updated documentation not to mention **id** already.

File: appliances/Trivago/ebb81aaf055247c19579723cc84b8e74.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    set_hostname: "$NAME"
+  cpu: '1'
+  graphics:
+    listen: 0.0.0.0
+    type: vnc
+  memory: '768'
+  os:
+    arch: x86_64
+  user_inputs:
+    ssh_user_key: "M|text|SSH KEY: Additional ssh key to add",
```
Comment (vholer) [inline appliances/Trivago/ebb81aaf055247c19579723cc84b8e74.yaml]: Please remove the trailing comma.

File: appliances/Trivago/ebb81aaf055247c19579723cc84b8e74.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+images:
+- name: FreeBSD-10.3
+  url: https://one-trivago.s3-website-us-west-1.amazonaws.com/freebsd-trv.qcow2
```
Comment (vholer) [inline appliances/Trivago/ebb81aaf055247c19579723cc84b8e74.yaml]: This URL (HTTPS) doesn't work for me, the HTTP-only works.

File: appliances/Trivago/ebb81aaf055247c19579723cc84b8e74.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  type: OS
+  dev_prefix: vd
+  driver: qcow2
+  size: 21474836480
+  checksum:
+    md5: 365b676bf858246e1b68f48f54159ba0
```
Comment (vholer) [inline appliances/Trivago/ebb81aaf055247c19579723cc84b8e74.yaml]: The checksum is wrong, I got:
md5: `8a49f2dac4ae37e25b366716d3a9c4cf`
sha256: `1a0dc8afbb11a1759d02949472b420c800e7e967aa3147d408862424b6cf552f`
Can you please check?

**Comment by perrymanuk** [comment]
@vholer Made the changes inside the file but I am not sure what you mean about renaming the file it self, I generated it originally with uuidgen and it looks lowercase from here ;)

**Comment by vholer** [comment]
Thank you, the appliance is now public!
http://marketplace.opennebula.org/appliance/ebb81aaf-0552-47c1-9579-723cc84b8e74

**Pull request merged**